plugins/org.eclipse.n4js.flowgraphs/src/org/eclipse/n4js/flowgraphs/N4JSFlowAnalyzerTestFeatures.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2017 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.flowgraphs;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.n4js.flowgraphs.model.ControlFlowEdge;
+import org.eclipse.n4js.flowgraphs.model.Node;
+import org.eclipse.xtext.xbase.lib.Pair;
+
+/**
+ * When enabled, the {@link N4JSFlowAnalyzerTestFeatures} class records data during creating and traversing the flow
+ * graphs. This recording is disabled by default for performance reasons.
+ */
+public class N4JSFlowAnalyzerTestFeatures {
+	static private boolean enabled = false;
+	static private List<Pair<Node, List<ControlFlowEdge>>> mergedEdges = new ArrayList<>();
```

Comment:
so, `>>>>` is new `}}}}}` which is new `))))))` ?

Reply:
It's the beauty of Generics.